### PR TITLE
remove most of the GPG email section

### LINKED ISSUE #1011: Broken link to EFF intro
@alanna-casey and @openprivacy 

On this page https://guidebook.civicactions.com/en/latest/common-practices-tools/security/encryption/, the link to https://ssd.eff.org/module/introduction-public-key-cryptography-and-pgp is going to 404 now in the below paragraph snippet.

> If you'd like some background on how encryption works, start by reading [An Introduction to Public Key Cryptography and PGP](https://ssd.eff.org/en/module/introduction-public-key-cryptography-and-pgp), a [Surveillance Self-Defense](https://ssd.eff.org/en) (also an excellent read) tutorial from your friends at the [Electronic Frontier Foundation](https://www.eff.org/) [[Donate!](https://supporters.eff.org/donate/)]

Do you have a recommendation for a suitable replacement?

### PR DESCRIPTION
**Pull request by openprivacy**
----
:books: Documentation preview :books:: https://civicactions-handbook--1055.org.readthedocs.build/en/1055/

### CODE REVIEW

**Review by dmundra — APPROVED** [review]
Looks great to me!

File: common-practices-tools/security/encryption.md [hunk 1]
```diff
@@ -1,186 +1,50 @@
...
-
-We recommend using both of these browser add-ons.
+We recommend using both of these browser add-ons that enhance privacy without encryption:
 
 -   [Privacy Badger](https://www.eff.org/privacybadger) blocks most ads and cookie collectors. Open source and free.
 -   [uBlock Origin](https://ublockorigin.com) is an ad content blocker. Open source and free.
 
-For search, we recommend changing your browser "default search" to [DuckDuckGo](https://duckduckgo.com/) or [Ecosia](https://www.ecosia.org/) as they protects privacy and avoids the filter bubble.
+For search, we recommend changin
+g your browser "default search" to [DuckDuckGo](https://duckduckgo.com/) or [Ecosia](https://www.ecosia.org/) as they protects privacy and avoid the filter bubble.
```
Comment (dmundra) [inline common-practices-tools/security/encryption.md]: Small typo here, the `g` is on a new line which is translating into a space on the site https://civicactions-handbook--1055.org.readthedocs.build/en/1055/common-practices-tools/security/encryption/#private-browsing.

**Review by dmundra — CHANGES_REQUESTED** [review]
Small typo fix needed.

**Comment by openprivacy** [comment]
@dmundra I see "This branch is out-of-date with the base branch" and an "Update branch" button. Should I push that? If so, merge or rebase?

**Comment by dmundra** [comment]
> @dmundra I see "This branch is out-of-date with the base branch" and an "Update branch" button. Should I push that? If so, merge or rebase?

@openprivacy, ya go for that and I recommend the default 'merge' option.

**Review by dmundra — APPROVED** [review]
Thanks for fixing the typo.

**Pull request merged**